services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/UpdateHankeITests.kt

```diff
@@ -599,9 +601,18 @@ class UpdateHankeITests(
         val createdHanke = hankeFactory.builder(USERNAME).withHankealue().save()
         val hankealue = createdHanke.alueet[0]
         assertThat(hankealue.nimi).isEqualTo("Hankealue 1")
+        val haittojenhallintasuunnitelma = createHaittojenhallintasuunnitelma()
         val request = createdHanke.toModifyRequest()
         val updatedRequest =
-            request.copy(alueet = listOf(request.alueet[0].copy(nimi = "Changed Name")))
+            request.copy(
+                alueet =
+                    listOf(
+                        request.alueet[0].copy(
+                            nimi = "Changed Name",
+                            haittojenhallintasuunnitelma = haittojenhallintasuunnitelma
```

Comment:
This test is about not updating anything except the name. I don't think it should be hijacked to test other changes. Maybe add another test?

Reply:
You're right. I added a new test.